src/Arcus.WebApi.Tests.Unit/Hosting/TestOpenIdServer.cs

```diff
@@ -138,7 +197,8 @@ private static void ConfigureServices(IServiceCollection services)
                             AllowedScopes = { "api1" }
                         }
                     })
-                    .AddDeveloperSigningCredential();
+                    //.AddDeveloperSigningCredential()
```

Comment:
Can be removed if we don't use it anymore.

Reply:
```suggestion
```